app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+
+	def new
+	end
+
+	def create
+		user = 	User.find_by(email: params[:session][:email])
+
+		if  user && user.authenticate(params[:session][:password])
+			session[:user_id] = user.id
```

Comment:
@nozomi-matsuoka Change this line to : 
`log_in user`
If you finished to add method in **SessionsHelper**

Reply:
I have a question.
Do you mean that the line 10(session[:user_id] = user.id) should replace with [log_in user] ?
I tried it, but there is an error.
<img width="803" alt="2018-09-06 19 28 22" src="https://user-images.githubusercontent.com/42955079/45154712-11ee1480-b20b-11e8-9c29-8b3cc0ffa990.png">